MdePkg/Include/Protocol/MediaSanitize.h

```diff
@@ -0,0 +1,147 @@
+/** @file

```

Comment:
might want to add tpl restrictions as the implementation of nvme definitely has assumptions it is being called from application or callback level

Reply:
Agreed. I will update.

Reply:
Could you please clarify your comment? Are you looking for comments stating the functions must be called from a certain TPL? Or are you looking for run time checks to ensure TPL is at callback or application?